cmd/lambda/leases/create.go

```diff
@@ -111,11 +99,46 @@ func CreateLease(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Start the step function to track usage for this lease
```

Comment:
TODO: This logic really belongs in the LeaseService layer, now that I look at it...